.gitignore

```diff
@@ -5,6 +5,8 @@
 *.Rproj
 .*.swp
 
+tests/gpu/misc/hosts
```

Comment:
It would be great if reproducible steps were described.
For example, why don't you delete the hosts file, and instead create a template file like `example.hosts` with a fake IP address, and explaining through a README that we need to create `hosts` file by copying the `example.hosts` file?

However, this has nothing to do with the build failures, so I think it is not too late to do this in another pull request.

Reply:
Great idea! Yeah, I can work on that for another pull request.